missionbit/settings.py

```diff
@@ -118,3 +140,9 @@
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
 STATIC_URL = '/static/'
+
+LOGIN_URL = 'http://localhost:8000/auth/login/google-oauth2/'
```

Comment:
There's probably a more robust way to generate this URL that will work even when running on a server or a different port.